src/PostProcesses/postProcess.ts

```diff
@@ -477,6 +490,71 @@ export class PostProcess {
         this.width = -1;
     }
 
+    private _createRenderTargetTexture(textureSize: { width: number, height: number }, textureOptions: RenderTargetCreationOptions, channel = 0) {
+        for (let i = 0; i < this._textureCache.length; i++) {
+            if (this._textureCache[i].width === textureSize.width &&
+                this._textureCache[i].height === textureSize.height &&
+                this._textureCache[i]._postProcessChannel === channel) {

```

Comment:
Should `_postProcessChannel` really be a member of `InternalTexture`? It seems it's only of use in the `PostProcess` class (and only in this method), so I would see more something like:
```javascript
private _textureCache: Array<{texture: InternalTexture, postProcessChannel: number}> = [];
```

Reply:
Totally, it's definitely only used in `PostProcess`